bin/cli.js

```diff
@@ -67,7 +67,13 @@ import i18nTransform from '../dist/transform.js'
         '.yaml': yamlConfigLoader,
         '.yml': yamlConfigLoader,
       },
-    }).load(program.opts().config)
+    })
+    let result
+    if (program.opts().config) {
+      result = await configReader.load(program.opts().config)
+    } else {
+      result = await configReader.search()
+    }
     config = result.config
```

Comment:
This is still possibly problematic as if there is no config file this will fail (`result.config` is then `undefined`). I've pushed a fix as `7.0.2`